Automattic/woocommerce-payments
# Add Affirm and Afterpay providers to additional payment method setup screen

Fixes #6274

#### Changes proposed in this Pull Request

- Updates the design of additional payment method setup screen to account both BNPL eligible and non-eligible merchants
- Splits the list of payment methods into two blocks 
   - Popular in your country
   - Buy Now, Pay Later



#### Testing instructions
1. [Setup server](https://github.com/Automattic/woocommerce-payments-server/pull/3405)
2. Disable additional payment methods 
![image](https://github.com/Automattic/woocommerce-payments/assets/24650518/e9216791-6eb5-465e-8e03-fcf7e4a87d9b)
3. Click Enable to get to the screen at question
4. Observe that Affirm and Afterpay are grouped separately 
![image](https://github.com/Automattic/woocommerce-payments/assets/24650518/00a471f1-2957-499e-83ec-9cbd9061c99a)
5. Select both BNPL methods and any local method, click continue and observe that they are properly selected
![image](https://github.com/Automattic/woocommerce-payments/assets/24650518/9a8fd599-17d9-4c47-a6a3-c91803f7f5ea)
6. Click "Go to payment settings" and observe that the same methods are enabled
7. Disable feature flag in the server (refer to 1st point)
8. Repeat step 2-3 
9. Observe that the screen doesn't have BNPL section

-------------------


**Post merge**
